api/all/src/main/java/io/opentelemetry/api/metrics/DoubleHistogramBuilder.java

```diff
@@ -32,6 +34,20 @@ public interface DoubleHistogramBuilder {
    */
   DoubleHistogramBuilder setUnit(String unit);
 
+  /**
+   * Set the explicit bucket buckets boundaries advice, which suggests the recommended set of
+   * explicit bucket boundaries for this histogram.
+   *
+   * @param bucketBoundaries The explicit bucket boundaries advice.
+   * @see <a
+   *     href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-advisory-parameter-explicitbucketboundaries">Explicit
+   *     bucket boundaries advisory parameter</a>
+   * @since 1.32.0
+   */
+  default DoubleHistogramBuilder setExplicitBucketBoundariesAdvice(List<Double> bucketBoundaries) {
+    return this;
```

Comment:
I think technically it's ok to break, a similar question with Go OTel API interfaces led to this: https://github.com/open-telemetry/opentelemetry-specification/pull/3695 

But I agree it's better to avoid the potential breakage by adding defaults (even if they're just no-ops).

Reply:
> I think technically it's ok to break, a similar question with Go OTel API interfaces led to this: https://github.com/open-telemetry/opentelemetry-specification/pull/3695

That's correct. And yes, our strategy historically has been to provide default implementations of new API methods. It doesn't alert alternative implementations with compile time errors, but those implementations should be following the release notes. Also, its not the end of the world if a new method is added an alternative implementation treats it as a no-op.